Cython/Includes/numpy/__init__.pxd

```diff
@@ -988,13 +988,13 @@ cdef inline object get_array_base(ndarray arr):
 # Cython code.
 cdef inline int import_array() except -1:
     try:
-        __pyx_import_array()
+        return __pyx_import_array()
```

Comment:
I don't think this is necessary. And note that the NumPy project owns this file now. We shouldn't apply changes on our side without pushing them there first.

Reply:
I've undone this - while I think it would have been clearer I don't think it's worth the effort of changing in two places